guides/source/6_0_release_notes.md

```diff
@@ -605,10 +605,68 @@ Please refer to the [Changelog][active-storage] for detailed changes.
 
 ### Deprecations
 
+*   Deprecate  `config.active_storage.queue` in favor of `config.active_storage.queues.analysis`
+    and `config.active_storage.queues.purge`.
+    ([Pull Request](https://github.com/rails/rails/pull/34838))
+
+*   Deprecate `ActiveStorage::Downloading` in favor of `ActiveStorage::Blob#open`.
+    ([Commit](https://github.com/rails/rails/commit/ee21b7c2eb64def8f00887a9fafbd77b85f464f1))
+
+*   Deprecate using `mini_magick` for generating image variants in favor of
```

Comment:
Tiny nit: I’d say “Deprecate using MiniMagick *directly*” or similar. You can still use MiniMagick, you just have to do it through ImageProcessing.